administrator/components/com_templates/views/template/tmpl/default_description.php

```diff
@@ -15,6 +15,6 @@
 	<?php echo JHtml::_('templates.thumbModal', $this->template->element, $this->template->client_id); ?>
 </div>
 <h2><?php echo ucfirst($this->template->element); ?></h2>
-<?php $client = JApplicationHelper::getClientInfo($this->template->client_id); ?>
```

Comment:
I'd prefer to just add the appropriate use statement on the top of the file. But I don't insist ;)